parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ParquetMetadata.java

```diff
@@ -41,6 +40,10 @@
 
   private static final ObjectMapper objectMapper = new ObjectMapper();
 
+  static {
+    objectMapper.configure(SerializationConfig.Feature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
Could you please explain why it is necessary?

Reply:
Without this configuration change on the objectMapper three tests in parquet-cascading3 fail with the following error:

  testReadPattern(org.apache.parquet.cascading.TestParquetTupleScheme): [namecp] could not build flow from assembly: [java.io.IOException: Could not read footer: java.lang.RuntimeException: org.codehaus.jackson.map.JsonMappingException: No serializer found for class org.apache.parquet.schema.LogicalTypeAnnotation$StringLogicalTypeAnnotation and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationConfig.Feature.FAIL_ON_EMPTY_BEANS) ) (through reference chain: org.apache.parquet.hadoop.metadata.ParquetMetadata["fileMetaData"]->org.apache.parquet.hadoop.metadata.FileMetaData["schema"]->org.apache.parquet.schema.MessageType["fields"]->java.util.ArrayList[0]->org.apache.parquet.schema.PrimitiveType["logicalTypeAnnotation"])]

The reason is: the logical types are no longer represented as enum, but a classed, and Jackson can't serialize empty classes because FAIL_ON_EMPTY_BEANS feature is enabled by default.

Reply:
I see. Thanks.
Could you please add some comments about that so others will also understand?

Reply:
Sure, no problem, I will.